Source/Scene/LabelCollection.js

```diff
@@ -213,46 +213,88 @@ define([
 
     function repositionAllGlyphs(label, resolutionScale) {
         var glyphs = label._glyphs;
+        var text = label._text;
         var glyph;
         var dimensions;
-        var totalWidth = 0;
+        //var totalWidth = 0;
```

Comment:
As a rule, we never leave commented out code in place so please delete this and other code you've left it.  See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide for this and other details regarding our codebase.